config/config.go

```diff
@@ -289,6 +337,10 @@ func (s *QueryResponse) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	if err := unmarshal((*plain)(s)); err != nil {
 		return err
 	}
+	if s.Expect.Regexp == nil {
+		s.Expect = MustNewRegexp("")
```

Comment:
Not being set and being the empty string are different, if there's handling here it should be in the probe rather than here.

Reply:
I deleted this IF statement to accept nil.